server/src/controllers/notes.js

```diff
@@ -12,6 +12,18 @@ class Notes {
     });
   }
 
+  static async createNote(req, res) {
+    const Note = await notes.create({
+      title: req.body.title,
```

Comment:
This looks Nice and works, But you can use destructuring for the req.body to simplify some more.

Reply:
Alright..thanks